### DISCUSSION TITLE
feat: UI scale settings - smaller text (especially on mobile)

### DISCUSSION THREAD

**Discussion by uberrice** [open-webui/open-webui#5912] [Ideas]
**Is your feature request related to a problem? Please describe.**
It's a bit frustrating having a lot of the sides of the screen taken up by margins on the mobile site - I'd also like to have smaller text. I cannot find any way to change these options. 

**Describe the solution you'd like**
Settings in interface settings to set text size and/or ui scale, margins on the side of the page (mainly an issue on mobile site). 

**Describe alternatives you've considered**
No clear alternatives other than somehow hacking around with the dpi settings. 

**Additional context**
If this is something you'd like implemented, but nobody has time for, I could give it a crack myself. I'm not too versed with frontend stuff, but I could give it a try.

**Comment by lvickyj** [comment]
Very desirable, I would probably include moving the icon for the messages to the top on mobile devices and screen ratios so it doesn't take up horizontal space
